adapters/lora-vertx/src/main/java/org/eclipse/hono/adapter/lora/impl/LoraProtocolAdapter.java

```diff
@@ -213,10 +213,9 @@ void handleProviderRoute(final RoutingContext ctx, final LoraProvider provider)
                     final String contentType;
                     if (payload.length() > 0) {
                         contentType = String.format(
-                                "%s%s%s",
+                                "%s%s",
                                 LoraConstants.CONTENT_TYPE_LORA_BASE,
-                                provider.getProviderName(),
-                                LoraConstants.CONTENT_TYPE_LORA_POST_FIX);
+                                provider.getProviderName());
```

Comment:
does this content-type make a JMS client to interpret the message as a ByteMessage?

Reply:
Yes.

Reply:
https://github.com/apache/qpid-jms/blob/f9ab180136c790a53b775c91b24e9339aca8deec/qpid-jms-client/src/main/java/org/apache/qpid/jms/util/ContentTypeSupport.java#L101-L107